source/onboard/mattermost-limits.rst

```diff
@@ -14,19 +14,21 @@ In addition to these limits, visit our pricing page to see a complete list of fe
 
    These limits do not apply to self-hosted deployments. 
 
-Mattermost Free limits
-----------------------
+Mattermost Cloud Free limits
+----------------------------
 
 **Platform limits**
 
-- Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
-- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived.
```

Comment:
@johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

Reply:
@cwarnermm the limit is now 1GB for Cloud as per this [GSheet](https://docs.google.com/spreadsheets/d/1mjXnrPuunAiE9QIcVldOvX2s6_Ih49uUs5OHoFwxzc8/edit#gid=1538757693)